src/utils/gatsby.js

```diff
@@ -0,0 +1,33 @@
+const _ = require("lodash")
+
+const CountRoles = [
+  "Products & Services",
+  "Research & Development",
+  "Advocacy",
+  "Network",
+]
+
+function countCategoriesOrganizations(categories) {
+  return _.chain(categories)
+    .reduce((accum, { id: catId, data }) => {
+      const parentId = data.Parent && data.Parent[0].id
+      const orgIds =
+        data.Organizations &&
+        data.Organizations.filter(
+          org => _.intersection(org.data.Role, CountRoles).length
```

Comment:
I think I'd have to `apply` it since we're comparing against an Array. Is this what you were thinking?

```js
Array.prototype.includes.apply(org.data.Role, CountRoles)
```